### PR TITLE
RDS Citations

### LINKED ISSUE #54: Contact Information
We need to add a footer detailing how to work with us on adding more data, telling us data is incorrect, or suggest features.

### PR DESCRIPTION
**Pull request by dgasmith**
Adds a bit of metadata from RDS (including @mattwelborn heroic work on adding DOIs). Also adds pointers on help, closes #54.

### CODE REVIEW

File: app/dash_apps/reaction_viewer/index.py [hunk 1]
```diff
@@ -286,6 +297,49 @@ def display_value(value):
...
+                citation = citation.replace("</em>", "*").replace(
+                    "<em>", "*"
+                )  # Italics 1
+                citation = citation.replace("</i>", "*").replace(
+                    "<i>", "*"
+                )  # Italics 2
+                citation = citation.replace("</b>", "**").replace("<b>", "**")  # Bold
+                citation = citation.replace(", ***", "*, **").replace(
+                    "***, ", "**, *"
+                )  # HTML vs Markdown corrections
```
Comment (mattwelborn) [inline app/dash_apps/reaction_viewer/index.py]: We need a `format_citation` method badly :(

File: app/dash_apps/reaction_viewer/index.py [hunk 2]
```diff
@@ -286,6 +297,49 @@ def display_value(value):
...
+                )
+
+            mqcas_help = dedent(
+                """
+                We are still filling in data and expanding these Webapps.
+                If you have features suggestions or bug reports please file them [here](https://github.com/MolSSI/QCArchiveWebapps/issues/new/choose)
+                or if you wish to expand the datasets, methods, or bases please open an issue
+                [here](https://github.com/MolSSI/MQCAS/issues/new/choose).
+                """
+            )
```
Comment (mattwelborn) [inline app/dash_apps/reaction_viewer/index.py]: Consider putting this in its own card so it doesn't have to update every time we change the dataset.

Comment (dgasmith) [reply app/dash_apps/reaction_viewer/index.py]: Yea, its easier this way for now (IMO). Its pretty lightweight compared to anything else.

File: app/dash_apps/reaction_viewer/index.py (hunk 1 above)
Comment (dgasmith) [reply app/dash_apps/reaction_viewer/index.py]: Yea :/

**Pull request merged**